Mathlib/AlgebraicGeometry/GammaSpecAdjunction.lean

```diff
@@ -467,6 +467,27 @@ theorem toOpen_unit_app_val_c_app' {X : Scheme.{u}} (U : Opens (PrimeSpectrum Γ
 
 end ΓSpec
 
+lemma toSpecΓ_ΓSpec_adjunction_homEquiv {X : Scheme.{u}} {B : CommRingCat} (φ : B ⟶ Γ(X, ⊤)) :
+    toSpecΓ B ≫ ((ΓSpec.adjunction.homEquiv X (op B)) φ.op).app ⊤ = φ := by
+  rw [AlgebraicGeometry.ΓSpec.adjunction_homEquiv]
```

Comment:
You should use `(Scheme.ΓSpecIso B).inv` instead of `toSpecΓ` (they are defeq) because the latter is in terms of locally ringed spaces. Maybe I should add a warning. And the proof becomes
```
example {X : Scheme.{u}} {B : CommRingCat} (φ : B ⟶ Γ(X, ⊤)) :
    (Scheme.ΓSpecIso B).inv ≫ ((ΓSpec.adjunction.homEquiv X (op B)) φ.op).app ⊤ = φ := by
  simp [Adjunction.homEquiv_apply]
  rw [ΓSpec.adjunction_unit_app_app_top]
  simp
```
Either you should squeeze it or maybe investigate why the simp lemma does not trigger.
Anyways in general you shouldn't unfold `ΓSpec.adjunction` at all and use the simp lemmas provided. It should be always possible as the morphism are uniquely identified by the adjunction properties.

Reply:
Ah I remember, you have told me before. We should just make all of the forbidden ones private.